entrypoint.sh

```diff
@@ -70,22 +70,31 @@ for namespace in $NAMESPACES; do
       label_selector="-l OWNER!=TILLER"
     fi
 
-    /kubectl --namespace="${namespace}" get --export -o=json "$type" $label_selector | jq --sort-keys \
+    kubectl --namespace="${namespace}" get "$type" $label_selector -o custom-columns=SPACE:.metadata.namespace,KIND:..kind,NAME:.metadata.name --no-headers | while read -r a b name; do
```

Comment:
Start indenting code here

Reply:
I ran https://github.com/mvdan/sh on it. There were some inconsistencies but it seemed 4 spaces was the most common.